src/payform.coffee

```diff
@@ -106,13 +106,13 @@
       type: 'visa'
       pattern: /^4/
       format: defaultFormat
-      length: [13, 16]
+      length: [13, 16, 19]
       cvcLength: [3]
       luhn: true
     }
     {
       type: 'mastercard'
-      pattern: /^(5[1-5]|2[2-7])/
+      pattern: /^(5[1-5]|677189)|^(222[1-9]|2[3-6]\d{2}|27[0-1]\d|2720)/
```

Comment:
This new Regex patterns addresses the BIN changes for Mastercard introduced in 2017:

https://www.mastercard.us/en-us/issuers/get-support/2-series-bin-expansion.html